python/ray/_private/log_monitor.py

```diff
@@ -440,6 +447,14 @@ def run(self):
                 time.sleep(0.1)
 
 
+def is_proc_alive(pid):
+    try:
+        return psutil.Process(pid).is_running()
+    except Exception:
```

Comment:
Maybe we should catch the OS Error?

Reply:
psutil raises `psutil.NoSuchProcess` if the PID is invalid, which does not inherit from OSError. I will change it to `psutil.NoSuchProcess`